index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FoodBlog</title>
+    <link rel="stylesheet" type="text/css" href="style.css"
+  </head>
+  <body>
+    <div class="header">
+      <h1 class="top-header">Food Blog</h1>
+    </div>
+    <div class="menu">
+      <a href="overview.html">Noodles</a>
+      <a href="overview.html">Rice</a>
+      <a href="overview.html">Meat</a>
+      <a href="overview.html">Vegi</a>
+    </div>
+    <div class="welcome">
+      <img src="./images/reisfeld.jpeg" alt="reisfeld" style="width: 100%" />
+      <div class="welcome-title">
+        <a href="overview.html">Welcome</a>
+      </div>
+    </div>
+    <div class="footer">
+      <a href="#aboutme">About Me</a>
+      <a href="#contact">Contact</a>
+    </div>
+    <h2 class="aboutme" id="aboutme">
+      <img src="./images/me.jpg" alt="me" style="width: 100%" />
+      <p>
+        I am from vietnam and a food lover. In our cuture, we love to have meal
+        with family and share all what we have on the table. I always looking
+        for good recipes to suprise my family. During my trip in 2016, I could
+        have a look in other kitchen around the world and could take a lot of
+        inspriration with me.
+      </p>
+    </div>
+    <div class="contact" id="contact">
```

Comment:
Because the `contact` class has the same properties as `aboutme`, you could remove `contact` and set here `aboutme` instead (maybe it would make sense to rename the class to something more general):
```suggestion
    <div class="aboutme" id="contact">
```